03-blog-preview/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- displays site properly based on user's device -->
+
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./assets/images/favicon-32x32.png"
+    />
+
+    <title>Frontend Mentor | Blog preview card</title>
+    <!-- link to the font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300..900;1,300..900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body class="yellow-background">
+    <div class="card">
+      <img
+        src="./assets/images/illustration-article.svg"
+        alt="illustration"
+        class="illustration"
+      />
+      <p class="yellow-background bold black-text">Learning</p>
+
+      <p class="small-p">Published 21 Dec 2023</p>
```

Comment:
[`time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) tag seems to be a good fit here.